src/Tribe/Views/V2/View.php

```diff
@@ -787,6 +787,9 @@ public function next_url( $canonical = false, array $passthru_vars = [] ) {
 				$url = add_query_arg( $query_args, $url );
 			}
 
+			// Remove the inverse of the page key we are using.
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 441593dbe34e42e31b9d91393ccea923</sub>